php/php.editor/src/org/netbeans/modules/php/editor/model/impl/OccurenceBuilder.java

```diff
@@ -460,7 +461,7 @@ void prepare(MethodDeclaration methodDeclaration, MethodScope scope) {
 
     void prepare(final MagicMethodDeclarationInfo node, MethodScope scope) {
         if (canBePrepared(node.getOriginalNode(), scope)) {
-            if (node.getKind().equals(Kind.METHOD)) {
+            if (node.getKind() == Kind.METHOD || node.getKind() == Kind.STATIC_METHOD) {
```

Comment:
Not sure if it could be useful to have some utility method like `...Utils.isMethod(Kind)` returning `true` for these 2 types.